packages/api-explorer/src/lib/parameters-to-json-schema.js

```diff
@@ -56,11 +56,14 @@ function getOtherParams(pathOperation, oas) {
           schema.type = 'array';
           schema.items = current.schema.items;
         }
-
         if (typeof current.schema.default !== 'undefined') schema.default = current.schema.default;
         if (current.schema.enum) schema.enum = current.schema.enum;
         if (current.schema.type) schema.type = current.schema.type;
         if (current.schema.format) schema.format = current.schema.format;
+        if (current.schema.pattern) schema.pattern = current.schema.pattern;
+        if (current.schema.minimum) schema.minimum = current.schema.minimum;
+        if (current.schema.maximum) schema.maximum = current.schema.maximum;
+        if (current.examples) schema.examples = current.examples;
```

Comment:
Why not add:
```
schema = {
  ...current.schema,
  ...schema,
  ...current.examples ? {examples: current.examples} : {}
}
```
(or with the inverse merge order)?

Reply:
We should add a test about this hotfix

Reply:
> Why not add:

I didn't use the spread operator, because I didn't know if everything we pass to today in the schema object is compatible.

Reply:
> We should add a test about this hotfix

ok